api/v1alpha3/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
Please use `go 1.16` for this repository, this autogenerated thing got introduced in `go 1.17`.